services/ui-src/src/components/tables/EntityRow.tsx

```diff
@@ -30,13 +40,20 @@ export const EntityRow = ({
     switch (entityType) {
       case ModalDrawerEntityTypes.TARGET_POPULATIONS:
         return !!entity?.transitionBenchmarks_applicableToMfpDemonstration;
+      case "initiative":
```

Comment:
Nit: Maybe we create a ModalOverlayEntityTypes enum to call this from? Never really liked the typing scheme for this but still

Reply:
definitely agree, going to capture this as part of a clean up ticket.